gemrb/plugins/CREImporter/CREImporter.cpp

```diff
@@ -710,6 +693,9 @@ void CREImporter::ReadChrHeader(Actor *act)
 		//fallthrough
 	case CREVersion::GemRB:
 		for (int i = 0; i < QSPCount; i++) {
+			// FIXME: we are reading 4 bytes then suspiciously discarding 3 of them
+			// I wonder if those the bytes are supposed to fill in the 3 byte holes we are leaving in QSlots
```

Comment:
Like I said, there's nothing suspicious here, it's just the way the data is stored. The QSlot gap (index offset) is for the hardcoded set of slots.